core/src/test/java/io/paradoxical/cassieq/unittests/server/TestService.java

```diff
@@ -9,8 +13,22 @@
 import lombok.Getter;
 
 import java.util.List;
+import java.util.concurrent.CountDownLatch;
 
 public class TestService extends ServiceApplication implements ModuleOverrider {
+
+    private final CountDownLatch runLatch = new CountDownLatch(1);
+
+    public void waitForRun() throws InterruptedException {
```

Comment:
some tests were failing because something wasn't hot and the test started before the api was running.